.github/workflows/tests.yml

```diff
@@ -103,10 +109,29 @@ jobs:
     - name: Run docutils CLI
       run: echo "test" | myst-docutils-html
 
+  # https://github.com/marketplace/actions/alls-green#why
+  check:  # This job does nothing and is only used for the branch protection
```

Comment:
This is what should eventually replace all the individual branch protection checks that belong to this workflow.